eth/downloader/peer.go

```diff
@@ -362,11 +367,12 @@ func (p *peer) String() string {
 	defer p.lock.RUnlock()
 
 	return fmt.Sprintf("Peer %s [%s]", p.id,
-		fmt.Sprintf("headers %3.2f/s, ", p.headerThroughput)+
-			fmt.Sprintf("blocks %3.2f/s, ", p.blockThroughput)+
-			fmt.Sprintf("receipts %3.2f/s, ", p.receiptThroughput)+
-			fmt.Sprintf("states %3.2f/s, ", p.stateThroughput)+
-			fmt.Sprintf("lacking %4d", len(p.lacking)),
+		fmt.Sprintf("hs %3.2f/s, ", p.headerThroughput)+
```

Comment:
I know this was here before but I suggest that you change this to just one call to `fmt.Sprintf`. I don't think this makes it either more readable or cleaner.